packages/dds/merge-tree/src/attributionCollection.ts

```diff
@@ -320,6 +325,7 @@ export class AttributionCollection implements IAttributionCollection<Attribution
 			if (i !== 0 || !areEqualAttributionKeys(lastEntry, other.keys[i])) {
 				// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 				this.offsets.push(other.offsets[i]! + this.length);
+				// TODO: see line 305
```

Comment:
hm?

Reply:
@jzaffiro, just calling your attention here - was merged like this